### ISSUE TITLE
Bitget got 403 Forbidden

### ISSUE THREAD

**Issue by joeky888** [PrivateRookie/ws-tool#35]
Binance example works fine, however, when I was trying to connect to bitget I got 403 forbidden.

Websocat works fine.

```fish
websocat wss://ws.bitget.com/mix/v1/stream # Using cli tool successfully connected, and then sending some subscribe messages

> {"op":"subscribe","args":[{"instType":"mc","channel":"ticker","instId":"BTCUSDT"},{"instType":"mc","channel":"candle1m","instId":"BTCUSDT"}]}
```
Curl with default http2 got 403 too.

```fish
curl https://ws.bitget.com/mix/v1/stream \
 -H "Upgrade: websocket" \
  -H "Sec-WebSocket-Version: 13" \
  -H "Connection: Upgrade" \
  -H "Sec-WebSocket-Key: Z29yYWRkdQo=" # Get 403 Forbidden
```

Curl with http 1.1 works fine.

```fish
curl --http1.1 https://ws.bitget.com/mix/v1/stream \
 -H "Upgrade: websocket" \
  -H "Sec-WebSocket-Version: 13" \
  -H "Connection: Upgrade" \
  -H "Sec-WebSocket-Key: Z29yYWRkdQo=" # Works
```

I suspect the reason is because of http2? But then I saw the issue https://github.com/PrivateRookie/ws-tool/issues/14
I have already tried actix-web, tokio-tungstenite and fastwebsockets, really need some help here, thank you.

**Comment by PrivateRookie** [comment]
Looks like ws-tool always specify port in `Host` req header, but cloudflare reject this kind of Host header, so we always got 403 reponse, I have make a hot fix for this issue, please try with master branch

**Comment by PrivateRookie** [comment]
Looks like ws-tool always specify port in Host req header, but cloudflare reject this kind of Host header, so we always got 403 reponse, I have make a [hot fix](https://github.com/PrivateRookie/ws-tool/commit/526f93a06ce0f9c0c11b82dfabf402767ea599ea) for this issue, please try with master branch.

Host header [spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Host)

**Comment by joeky888** [comment]
I can confirm this hot fix works!

**Comment by PrivateRookie** [comment]
ok, fix version 0.7.5 has been published.

**Comment by joeky888** [comment]
Nicely done! This project should get more attention! I can finally get rid of Golang.
